grammars/sql.cson

```diff
@@ -29,7 +31,11 @@
         'name': 'keyword.other.create.sql'
```

Comment:
I presume you meant the instance of `create.other.create.sql` on line 39 should be `create.other.drop.sql`. The instance on line 18 is apart of the pattern for the `create` statement, so that should stay.